examples/example-cfep-3.yml

```diff
@@ -1,5 +1,6 @@
 action: cfep3_copy
 anaconda_org_packages:
-  # dict of (sha256, channel[/label/<label>]/subdir/filename.extension)
-  cd11f1f0fbe8c203212d002ca4e01795d0b1c886ce47d8b08e6bca3366a7c8e6: jaimergp/osx-arm64/conda-standalone-24.7.1-hce30654_0.conda
+  # list of {package: channel[/label/<label>]/subdir/filename.extension, sha256: SHA256}
```

Comment:
Doesn't conda-build use a `::` syntax for this? Maybe we should use that?

Reply:
By this I mean prepending a package with a channel name.

Reply:
This is the "path" in the anaconda.org server. e.g. concatenating this to `conda-web.anaconda.org` gives you the artifact. I'm using conda-web because of the labels; for `main` packages conda.anaconda.org would work too.

The `::` syntax is technically for MatchSpecs only, and this is more of a PackageRecord. Not that it matters much:

- jaimergp/label/something/noarch/package-1.2.3-0.tar.bz2
- jaimergp/label/something/noarch::package-1.2.3-0.tar.bz2

One more character and the opportunity for YAML to parse that as a weird dict :P I don't see much of advantage, really.